src/jit/rationalize.cpp

```diff
@@ -605,6 +605,9 @@ void Rationalizer::RewriteAssignment(LIR::Use& use)
             }
             JITDUMP("Rewriting GT_ASG(%s(X), Y) to %s(X,Y):\n", GenTree::NodeName(location->gtOper),
                     GenTree::NodeName(storeOper));
+#if NODEBASH_STATS
+            GenTree::RecordOperBashing(storeBlk->OperGet(), storeOper);
+#endif
             storeBlk->gtOper = storeOper;
             storeBlk->gtFlags &= ~GTF_DONT_CSE;
```

Comment:
Actually SetOper now does a whole bunch of stuff it didn't do originally so there are probably good reasons not to call it in places; I'll just change those to ChangeOperUnchecked() I think.